.github/workflows/build_cmake.yml

```diff
@@ -27,7 +27,7 @@ jobs:
           }
         - {
             name: "Ubuntu Build", artifact: "Linux.tar.xz",
-            os: ubuntu-18.04,
+            os: ubuntu-latest,
```

Comment:
For both of these, is `ubuntu-latest` appropriate? I assumed so but we can also update it to `ubuntu-20.04` or `ubuntu-22.04`.

Reply:
Actually, I thought about it and probably the smallest version bump here is more desirable so I updated these to `ubuntu-20.04`.